### DISCUSSION TITLE
GitHub v4 API Unreliable

### DISCUSSION THREAD

**Discussion by unformatt** [community/community#24532] [Apps, API and Webhooks]
<p>I have an app that previously used the v3 API and I recently switch to the v4 API. The v4 API seems much less reliable. I get errors daily that cause me to have to retry requests. It’s understandable when the API admits fault e.g. <code>Something went wrong while executing your query.</code>. But today I’m seeing the API tell me that repos don’t exist, which is not true. I’m seeing this across multiple repos and mulitple API keys. For hours, it’s been randomly returning <code>Could not resolve to a Repository with the name 'xxx/xxx</code>. Github status says everything is fine.</p>

**Comment by unformatt** [comment]
<p>Still going on. No one else having API problems?</p>

**Comment by mikimichaeli** [comment]
<p><a class="mention" href="/unformatt">@unformatt</a> We are also experiencing the exact same issue for the past 12 hours. This needs serious attention!</p>

**Comment by unformatt** [comment, marked as answer]
<p>I contacted support directly and the missing repo error is fixed now.</p>

**Comment by Ghostridr** [comment]
If this is due to a recent update to GitHub Desktop, where my issue is, it is telling me I have no repo's. So I have to re-add their locations, create new, or clone all my repositories all over again. However, I had made a plethora of changes that have yet to be committed to GitHub. So now I question whether or not I should bother trying to keep using the desktop application since I don't know if it's going to cause changes to be lost or not merge correctly and then I'll have an even bigger problem finding out what went wrong between the commits.

**Reply by Ghostridr** [reply]
Currently it's only showing that I have 1 repo, which I just created.
